### PR TITLE
Lower MaxValidUntilBlockIncrement limit

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo/issues/2026 https://github.com/neo-project/neo/issues/2027

### CODE REVIEW

File: src/neo/SmartContract/ApplicationEngine.Blockchain.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ namespace Neo.SmartContract
 {
     partial class ApplicationEngine
     {
-        public const uint MaxTraceableBlocks = Transaction.MaxValidUntilBlockIncrement;
+        public const uint MaxTraceableBlocks = 200_000;
```
Comment (erikzhang) [inline src/neo/SmartContract/ApplicationEngine.Blockchain.cs:11]: Transactions should be kept for at least one year.

**Review by Tommo-L — APPROVED** [review]

**Review by erikzhang — APPROVED** [review]

**Comment by shargon** [comment]
Move `MaxTraceableBlocks` to `protocol`  first?

**Comment by Tommo-L** [comment]
> Move `MaxTraceableBlocks` to `protocol` first?

Ping

**Comment by superboyiii** [comment]
> Move `MaxTraceableBlocks` to `protocol` first?

Agree.

**Pull request closed without merging**